apibindings/go/pkg/osvdev/osvdev.go

```diff
@@ -0,0 +1,295 @@
+package osvdev
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"math/rand/v2"
+	"net/http"
+	"time"
+
+	"github.com/ossf/osv-schema/bindings/go/osvschema"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	QueryBatchEndpoint = "/v1/querybatch"
+	QueryEndpoint      = "/v1/query"
+	GetEndpoint        = "/v1/vulns"
+
+	// DetermineVersionEndpoint is the URL for posting determineversion queries to OSV.
+	DetermineVersionEndpoint = "/v1experimental/determineversion"
```

Comment:
Should DetermineVersion stuff belong in `osvdevexperimental`?

Reply:
I'm thinking probably not? If we do it has to be detached from the current OSVClient, and I want OSVClient to match the OSV API 1:1.